src/analysis/special_functions/arsinh.lean

```diff
@@ -0,0 +1,190 @@
+/-
+Copyright (c) 2020 James Arthur. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Arthur, Chris Hughes, Shing Tak Lam.
+-/
+import analysis.special_functions.trigonometric
+import data.real.basic
+import analysis.special_functions.pow
+
+open real
+local attribute [pp_nodot] real.log
+noncomputable theory
+
+/-!
+# Inverse of the sinh function
+
+In this file we prove that sinh is bijective and hence has an
+inverse, arsinh.
+
+## Main Results
+
+- `sinm_injective`: The proof that `sinm` is injective
```

Comment:
mathlib docstrings usually use `*` instead of `-` for lists

Reply:
I'm pretty sure I've seen both, it's not a big deal I think.

Reply:
Okay, sorry for the spam in that case